luke-lukemichaeleisenberg/hw_2/3-dice-lab/js/app.js

```diff
@@ -0,0 +1,35 @@
+/*
+
+Creating a page where every time the page is loaded, the console logs random values for two dice.
+
+1) Write down pseudocode for the program described below.
+
+2) Follow the steps in your pseudocode to write the code for your function.
+* generate a random number between 1 - 6 and store to a variable, firstDie
+* generate a random number between 1 - 6 and store to a variable, secondDie
+* log the values of the first and second dice to the console, labeling each value
+* log the total of the two dice to the console, with a label
+
+3) Create a statement that runs the function you created whenever the page loads.
+  (Hint: Reference when the page loads with the code window.onload, and set its value to the name of the function you created, without parentheses.)
+
+*/
+
+//create a function declaration
+//use the random() function to get the numbers and store them to variables
+//log out the values
+//add the values together
+//log out the total
+//call the function when the window loads
+
+function rollDice()
+{
```

Comment:
Write your code for other people. 
Write for what they will expect, and remember the computer is not smart; it will do exactly what you tell it to do, and nothing more. Consider what this code returns:

```function foo()
{
  return
  {
    bar: 1
  };    
};

console.log(foo()); // => undefined
```
Egyptian style brackets are not just a stylistic preference, they are indeed a functional syntax.